webdriver-spec.html

```diff
@@ -3407,24 +3419,32 @@
  <li><p><a>Handle any user prompts</a> and return its value if it is an <a>error</a>.
 
  <li><p>If the <a>remote end</a> does not support
-  the <a>Set Window Size</a> <a>command</a> for
+  the <a>Set Window Dimensions</a> <a>command</a> for
   the <a>current top-level browsing context</a> for any reason,
   return <a>error</a> with <a>error code</a> <a>unsupported operation</a>.
 
- <li><p>If the properties <code>width</code> or <code>height</code>
-  are missing from the <var>parameters</var> argument,
-  or they are not integers, or less than 0 or greater than 2<sup>64</sup> – 1,
+ <li><p>If the properties <code>width</code> or <code>height</code> or
```

Comment:
This I will fix up in a separate PR as there are multiple instances through the spec like this.